database/createDatabase.sql

```diff
@@ -1,5 +1,5 @@
-DROP DATABASE IF EXISTS tmf_app_manager;
+-- DROP DATABASE IF EXISTS tmf_app_manager;
```

Comment:
Yeah -- it wasn't usually dropping anyway -- it just paused for ages while it tried, but then didn't because it still had active connections. So I just took this out to speed up the process. 

I think I only had the `DROP DATABASE` command because I thought the next `CREATE DATABASE` command would give an error if it already existed, but it doesn't, so probably don't need it. Just DROPping public schema seems to be enough for now. Might be a good idea to leave it commented though, as a reminder to turn it back on if we start doing more drastic things during the init?

Reply:
Ok. Good to know that the CREATE DATABASE will recreate one. I wasn't aware either.
Let's leave this comment for now.